docs/modules/reference/pages/service-assurance/monitors/AvailabilityMonitor.adoc

```diff
@@ -1,5 +1,6 @@
 
 = AvailabilityMonitor
```

Comment:
Maybe we flag this page to get removed at some point, since it's a long deprecated function?

Reply:
I could create a ticket to remove it now.